reconcile/change_owners/changes.py

```diff
@@ -236,8 +234,31 @@ def create_bundle_file_change(
             fileref=fileref,
             old=old_file_content,
             new=new_file_content,
+            old_content_sha=old_content_sha,
+            new_content_sha=new_content_sha,
             diffs=diffs,
         )
+
+    if old_content_sha != new_content_sha or old_path != new_path:
+        # there were no diffs but the content of the file has changed, e.g.
+        # - the SHA of the file: the SHA can change even when no diffs are detected.
+        #   this can be happen in case something undetectable has changed, e.g. comments
+        #   in a YAML file are invisible to YAML parsers
+        # - the path of a file has changed a.k.a. the file was moved. in such a case
```

Comment:
is list reordering also part of these 'invisible' change ? If so, it may be worth adding in this comment.

Reply:
i added a comment